go/def.bzl

```diff
@@ -153,11 +161,13 @@ def emit_go_asm_action(ctx, source, out_obj):
       " ".join(args),
   ]
 
+  f = _tmp_cmd_file(cmds, ctx, out_obj + "GoAsmCompileFile")
```

Comment:
Could you name the temp file `out_obj + ".GoAsmCompile.params"` to be consistent to follow convention in Bazel?